source/uri-options/uri-options.rst

```diff
@@ -280,6 +284,11 @@ pertaining to URI options apply here.
      - no
      - Relax TLS constraints as much as possible (e.g. allowing invalid certificates or hostname mismatches); drivers must document the exact constraints which are relaxed by this option being true
 
+   * - tlsDisableOcspEndpointCheck
+     - "true" or "false"
+     - Non-stapled OCSP is enabled
+     - Yes
+     - Disable non-stapled OCSP
```

Comment:
For those not as familiar with the OCSP spec, it might be clearer to spell out exactly what happens here; maybe something like "Certificates will not be checked against OCSP endpoints even if they are not in the cache"

Reply:
True: I've updated the wording. I've left out mention of the cache, instead going for the more general "if needed", and added a link that explains what "if needed" means